src/Controller.js

```diff
@@ -80,20 +95,76 @@
     },
 
     renderMessage(message) {
-      const messageElement = document.createElement('div');
-      messageElement.id = 'message';
+      const messageElement = document.createElement("div");
+      messageElement.id = "message";
       messageElement.innerHTML = message;
 
-      const viewport = document.querySelector('viewport');
+      const viewport = document.querySelector("#viewport");
       viewport.appendChild(messageElement);
 
       setTimeout(() => {
         viewport.removeChild(messageElement);
       }, 2000);
     },
+
+    portUpdate() {
+      const ship = this.ship;
+
+      const currentPortElement = document.getElementById("current-port");
+      const nextPortElement = document.getElementById("next-port");
```

Comment:
Declaring these `const`s improves readability of your code, well done! You can think if you could do the same in other places ;)